submissions/dummies/estimator.py

```diff
@@ -0,0 +1,56 @@
+import os
+import pandas as pd
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.compose import make_column_transformer
+from sklearn.preprocessing import OrdinalEncoder
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+
+def _merge_external_data(X):
+    filepath = os.path.join(os.path.dirname(__file__),
+                            'external_data.csv')
+    data_weather = pd.read_csv(filepath)
+    X_weather = data_weather[['Date', 'AirPort', 'Events']]
+    X_weather = X_weather.rename(
+        columns={'Date': 'DateOfDeparture', 'AirPort': 'Arrival'})
+    X_merged = pd.merge(
+        X, X_weather, how='left', on=['DateOfDeparture', 'Arrival'], sort=False
+    )
+    return X_merged
+
+
+def _encode_dates(X):
+    # to avoid SettingwithCopyWarning
+    X_encoded = X.copy()
+    X_encoded['DateOfDeparture'] = pd.to_datetime(X['DateOfDeparture'])
+    X_encoded['year'] = X_encoded['DateOfDeparture'].dt.year
+    X_encoded['month'] = X_encoded['DateOfDeparture'].dt.month
+    X_encoded['day'] = X_encoded['DateOfDeparture'].dt.day
+    X_encoded['weekday'] = X_encoded['DateOfDeparture'].dt.weekday
+    X_encoded['week'] = X_encoded['DateOfDeparture'].dt.week
+    X_encoded['n_days'] = X_encoded['DateOfDeparture'].\
+        apply(lambda date: (date - pd.to_datetime("1970-01-01")).days)
+    return X_encoded
+
+
+def get_estimator():
+    merge_transformer = FunctionTransformer(_merge_external_data,
```

Comment:
```suggestion
    merge_transformer = FunctionTransformer(
        _merge_external_data, validate=False
    )
```

Reply:
I am not sure that we need `validate=False` since this is the default in the last scikit-learn

Reply:
hmm then we must specify sklearn version in the requirements?

Reply:
yes, we should update the requirements to have the last version of scikit-learn and pandas.